modules/runner/src/main/ts/runner/TestRun.ts

```diff
@@ -92,6 +106,23 @@ export const runTests = (tests: Test[], state: RunState, actions: RunActions, re
   return loop(tests, (test) => runTest(test, state, actions, reporter));
 };
 
+const fakeFailure = (reporter: RunReporter, suite: Suite, state: RunState, siblings: string[], actions: RunActions) => {
+  const tests = suite.tests.length === 0 ? suite.suites[0].tests : suite.tests;
+  const filename = tests[0]?.file || 'Unknown';
```

Comment:
Could we use a ternary here we try to avoid the `&&` and `||` hacks. I guess we have this style in other places in the project but should we still try to follow what we do in regular editor development?